Druid-Tests/DRDispatchTableBuilderTest.class.st

```diff
@@ -13,7 +13,7 @@ Class {
 { #category : #running }
 DRDispatchTableBuilderTest >> compilationUnitForTesting [
 
-	^ DRPrimitiveCompilationUnit new
+	^ DRCogitCompilationUnit new
```

Comment:
No, this should be a compilation unit, dot.
Cogit is a backend detail.

We should rename it later.